hangman.py

```diff
@@ -1,213 +1,304 @@
+    guesses = 6
+    letters_guessed = []
+    vowels = ('a', 'e', 'i', 'o', 'u')
+    warnings = 3
+    while 1:
+        if is_word_guessed(secret_word, letters_guessed):
+            print(victory(guesses, secret_word))
+            break
+        elif guesses <= 0:
+            print(f'\033[7mYou lost. Guesses exceeded. The secret word was - {secret_word}\033[0m')
+            break
+        else:
+            print(get_guessed_word(secret_word, letters_guessed))
+            print(f'You have {guesses} guesses left.')
+            available = get_available_letters(letters_guessed)
+            print(f'Available letters: {available}')
+            l_c = letter_check(guesses, warnings)
+            letter = l_c[0]
+            guesses = l_c[1]
+            warnings = l_c[2]
+            letters_guessed.append(letter.lower())
+            if letter.lower() in secret_word:
+                print('\033[1mRight guess!\033[0m')
+            elif letter.lower() in vowels and letter.lower() not in secret_word:
+                print('\033[31mWrong guess(\033[0m')
+                guesses -= 2
+            else:
+                print('\033[31mWrong guess(\033[0m')
+                guesses -= 1
+            print('--------------------------------------------')
+
+
+def match_with_gaps(my_word, other_word):
+    '''
+    my_word: string with _ characters, current guess of secret word
+    other_word: string, regular English word
+    returns: boolean, True if all the actual letters of my_word match the
+        corresponding letters of other_word, or the letter is the special symbol
+        _ , and my_word and other_word are of the same length;
+        False otherwise:
+    '''
+    my_word_new = my_word.replace(' ', '')
+    if len(my_word_new) == len(other_word):
+        for i in range(len(my_word_new)):
+            if my_word_new[i] == other_word[i]:
+                continue
+            elif my_word_new[i] == "_" and other_word[i] not in list(my_word_new):
+                continue
+            else:
+                return False
+        return True
+    else:
+        return False
+
+
+def show_possible_matches(my_word):
+    '''
+    my_word: string with _ characters, current guess of secret word
+    returns: nothing, but should print out every word in wordlist that matches my_word
+             Keep in mind that in hangman when a letter is guessed, all the positions
+             at which that letter occurs in the secret word are revealed.
+             Therefore, the hidden letter(_ ) cannot be one of the letters in the word
+             that has already been revealed.
+    '''
+    my_word_new = my_word.replace(' ', '')
+    possible_matches = []
+    for word in wordlist:
+        if match_with_gaps(my_word_new, word):
+            possible_matches.append(word)
+
+    if possible_matches is []:
+        print('No matches founded')
+    else:
+        print(' '.join(possible_matches))

```

Comment:
I believe there should be different output

Reply:
What then?

Reply:
'Possible word matches are:'
Please - try implement your output to be identical to the examples in requirements (all examples)